Resources/config/css.yml

```diff
@@ -89,6 +89,7 @@ system:
                 - 'bundles/ezplatformui/css/modules/selection-filter.css'
                 - 'bundles/ezplatformui/css/modules/button.css'
                 - 'bundles/ezplatformui/css/modules/table-data.css'
+                - 'bundles/ezplatformui/css/modules/breadcrumbs-list.css'
                 - 'bundles/ezplatformui/css/modules/breadcrumbs.css'
```

Comment:
ah we already had a breadcrumbs.css module ? so I don't get why you are adding breadcrumbs-list.css, aren't all our breadcrumbs based on a list ?

The point here is to avoid having n times the same (or same-ish) CSS for the same thing. So this PR is an opportunity to unify the CSS (and HTML) code of our breadcrumbs to use the same thing and only keep the very specific CSS out of the breadcrumb module.
So you should check the breadcrumb in the LocationView, the ones in the Locations and Related Content tabs and the ones in the admin panel (like when viewing a Section) so that they use the breadcrumbs.css module for most of their style.